### PR TITLE
Task #160334 chore: Add isApproved method

### PR DESCRIPTION
**Pull request by deepgawade89**

### CODE REVIEW

File: src/com_tjvendors/site/includes/vendor.php [hunk 1]
```diff
@@ -560,6 +561,18 @@ public function getPhoneNumber()
...
 	}
 
+	/**
+	 * Get the vendor zip
+	 *
+	 * @return  String
+	 *
+	 * @since   __DEPLOY_VERSION__
+	 */
+	public function getApproved()
```
Comment (snehal0904) [inline src/com_tjvendors/site/includes/vendor.php:571]: @deepgawade89 getApproved - isApproved

Comment (deepgawade89) [reply src/com_tjvendors/site/includes/vendor.php]: Resolved

File: src/com_tjvendors/site/includes/vendor.php [hunk 2]
```diff
@@ -560,6 +561,18 @@ public function getPhoneNumber()
 		return $this->phone_number;
 	}
 
+	/**
+	 * Method return vendor approval status
+	 *
+	 * @return  String
```
Comment (manojLondhe) [inline src/com_tjvendors/site/includes/vendor.php:567]: string or boolean?

Comment (deepgawade89) [reply src/com_tjvendors/site/includes/vendor.php]: Updated. It's boolean

**Pull request merged**